{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -1,13 +1,17 @@
+# This is the Dockerfile that builds a fully functional image of your app
 {% include 'misc/header.py' %}
 ARG DEPENDENCIES_VERSION=latest
+# Get the previously built image(Docker.base) with pip dependencies and extend it
```

Comment:
I would add some comment about that it is done like this in order to speed up repeated builds.